controllers/v2/observationsController.js

```diff
@@ -593,13 +614,19 @@ module.exports = class Observations extends v1Observation {
                 let solutionDocumentFieldList = await observationsHelper.solutionDocumentFieldListInResponse()
 
                 response.result.solution = await _.pick(solutionDocument, solutionDocumentFieldList);
+                response.result.program = programDocument;
 
                 let submissionDocument = {
                     entityId: entityDocument._id,
                     entityExternalId: (entityDocument.metaInformation.externalId) ? entityDocument.metaInformation.externalId : "",
                     entityInformation: entityDocument.metaInformation,
                     solutionId: solutionDocument._id,
                     solutionExternalId: solutionDocument.externalId,
+                    programId : programDocument._id,
+                    programExternalId : programDocument.externalId,
+                    programInformation : {
```

Comment:
@aman-tunerlabs - Missing isAPrivateProgram and omit the same in programInformation